doc/API_Doc/ApiGlobe.js.html

```diff
@@ -33,7 +33,7 @@ <h1 class="page-title">Source: ApiGlobe.js</h1>
  */
 
 
-define('Core/Commander/Interfaces/ApiInterface/ApiGlobe', [
+define( [
```

Comment:
This is an artifact of the `sed` script being used to transform "named defines" to "unnamed defines" prior to running `amdtoes6` (which doesn't support named defines). The `sed` script modified that file, but `amdtoes6` obviously didn't.

IMO, either we:
 * ignore it, leave the commit like that; we'll eventually remove the generated doc from `master` anyway
 * remove `doc` from `master` first, then rebase that change on top
 * exclude that file from the commit